state/allwatcher.go

```diff
@@ -101,6 +101,8 @@ func makeAllWatcherCollectionInfo(collNames ...string) map[string]allWatcherStat
 			collection.docType = reflect.TypeOf(backingRemoteApplication{})
 		case applicationOffersC:
 			collection.docType = reflect.TypeOf(backingApplicationOffer{})
+		case generationsC:
```

Comment:
Are we still calling this `generations`? Are we going to rename it to branhes?

Reply:
As part of the (likely 2nd of 3) reworks for the design change a lot of the nomenclature was changed to refer to uncommitted generations as "branches", but after one discussion with @jameinel we stuck the collection name as once committed, it still becomes the latest model "generation". I am happy to change it - we are after all departing significantly from that view of the world.